### DISCUSSION TITLE
Sorry if this is the wrong place to ask this, but how do I make the ending bit of this sound less choppy?

### DISCUSSION THREAD

**Discussion by The-eater-of-flesh** [stakira/OpenUtau#1307] [General]
I'm making a Kasane Teto cover of Chonny Jash's cover of The Mind Electric, and I'm running into a few issues due to the fact that I'm trying to make a Japanese voicebank speak English. It sounds okay-ish, except for that "...light have caused some glare" sounds really choppy. Does anyone know how to fix this?

https://github.com/user-attachments/assets/dd201de5-9da4-40dc-bdd9-6e49304955e6

**Comment by BagelHero** [comment]
So, we actually have a more user friendly way of handling this. There's a Phonemizer called `EN to JA` that will allow you to write lyrics in plain English, and the output will be Japanese phonemes. You might want to try that? I think even with heavy edits, it will be easier to get a more fluent sounding result.

**Reply by The-eater-of-flesh** [reply]
My apologies for how late I'm replying, but I literally CANNOT find the en to ja phonemizer.. halp meh

**Reply by The-eater-of-flesh** [reply]
foudn it but it sounds like an animal crossing character?
